inc/saiacl.h

```diff
@@ -816,7 +816,7 @@ typedef enum _sai_acl_table_attr_t
     /**
      * @brief Range type defined
      *
-     * @type sai_acl_range_type_t
+     * @type sai_s32_list_t sai_acl_range_type_t
```

Comment:
Yes, you can put any item from sai_acl_range_type_t :

    /** L4 Source Port Range */
    SAI_ACL_RANGE_L4_SRC_PORT_RANGE,

    /** L4 Destination Port Range */
    SAI_ACL_RANGE_L4_DST_PORT_RANGE,

    /** Outer Vlan Range */
    SAI_ACL_RANGE_OUTER_VLAN,

    /** Inner Vlan Range */
    SAI_ACL_RANGE_INNER_VLAN,

    /** Packet Length Range in bytes */
    SAI_ACL_RANGE_PACKET_LENGTH

So currently there can be up to 5 items